### DISCUSSION TITLE
Build Always Failed when Preparing build machine

### DISCUSSION THREAD

**Discussion by bevanchristian** [codemagic-ci-cd/codemagic-docs#1964] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

iOS (Swift, Objective-C)

### Steps to reproduce

run my schema and its always failed

even in the master branch where yml is working it doesn't work

<img width="826" alt="Screen Shot 2023-08-11 at 19 25 56" src="https://github.com/codemagic-ci-cd/codemagic-docs/assets/50088801/919d0e91-d506-48ef-9871-aeca6767f04c">

and there is no log 

<img width="822" alt="Screen Shot 2023-08-11 at 19 29 13" src="https://github.com/codemagic-ci-cd/codemagic-docs/assets/50088801/ecf3b31a-85c1-433f-9921-f6a8af3d01c3">

and i retry it until 10 times its always failed in 2 or 3 second when preparing device 

### Expected results

 Preparing build machine success 

### Actual results

 Preparing build machine always failed

### Build id (optional)

64d628c4bf2afda852cb9745

**Comment by kulakarla** [comment]
Hello, we are aware of the issue and are investigating it.
